pages/_tutorials/dde/howto_create_new_profile.md

```diff
@@ -0,0 +1,114 @@
+    url: https://bioschemas.org/people/RicardoArcila
+  - "@type": Person
+    name: "Kenneth McLeod"
+    "@id": https://bioschemas.org/people/KennethMcLeod
+    url: https://bioschemas.org/people/KennethMcLeod
+  dateModified: 2022-11-29
+  description: "In this how-to, we will guide you through the necessary steps to create a new Bioschemas profile. The next tutorial will tell you how to add it to the DDE."
+  keywords: "schema.org, markup, structured data, bioschemas profile"
+  license: CC-BY 4.0
+  version: 2.0
+---
+
+## 1. Before you start
+
+Profiles are developed within [working groups](/groups). Before starting on a new profile, the Working Group Leads should create a new issue on the [GitHub issue tracker](https://github.com/BioSchemas/specifications/issues) to discuss with the Steering Council the need for the new profile. This will require providing use cases that the profile is aimed to satisfy. For full details, please see the [Bioschemas Governance document](https://github.com/Bioschemas/governance/blob/master/governance.md#proposing-a-new-profile).
+
+### 1.1 Use Case Development
+
+The use cases should be captured in a Markdown file that can be added to the [use case](/useCases) collection. Bioschemas use cases tend to focus on search scenarios, but some also support lightweight data exchange.
+
+A good use case will be specific about what is needed as input and what is expected as output. For example, in the following pattern you would fill in specicifics for the `X`, `Y`, and `Z`.
+
+> Alicia is looking for an `X` by using terms `Y`. To enable Alicia to distinguish between returned resources, properties `Z` are provided in the search results.
+
+The use cases are helpful to identify the properties required and what cardinality level they should be within the profile. As a rule of thumb, __search terms__ are at the Minimal Level, __diambiguation properties__ are at the Recommended Level, and __other properties of interest__ are provided at the Optional Level.
+
+### 1.2 Cross-walk over Existing Terminologies
+
+There are many existing terminologies within the life sciences. The aim of Bioschemas is not to replace these, but to provide a lightweight set of terms that support the discovery of resources on the Web. The Bioschemas types and properties should be inspired by and reuse as much as possible the existing terminologies from the domain.
+
+The attributes that have been identified through the development of the use cases should be captured into a Crosswalk GSheet ([template](https://docs.google.com/spreadsheets/d/1UrEWtP8-ezbhG8anmNgaDwWbCUAi4e7cqq31dr69yf0/edit?usp=sharing)) within the Bioschemas Specification GDrive [folder](https://drive.google.com/drive/folders/0Bw_p-HKWUjHoNThZOWNKbGhOODg?resourcekey=0-H1gdWdmBSIZ1Nj6LGxuj9Q&usp=share_link). The first three columns allow for capturing the property, a description of the property, and the expected type of the values for the property.
```

Comment:
Does it make sense to add a fourth column Expected carnality the crosswalk template?

Reply:
This is really a question of how much do we want to capture in the crosswalk. It could be argued that having the expected types is too much information.

I think that the name and description are absolutely required as they are what define the property. I'd included expected type since at the time of creating I thought that also was part of the definition of the property but on reflection I think it could be dropped.

I believe that we want to keep the crosswalk clear of profile design aspects, so I would not include cardinality or marginality.

Perhaps we first try with just name and description and see where how that goes.